pkg/crc/ssh/ssh.go

```diff
@@ -25,6 +27,13 @@ func (runner *SSHRunner) Run(command string) (string, error) {
 	return runner.runSSHCommandFromDriver(command, false)
 }
 
+func (runner *SSHRunner) SetTextContentAsRoot(destFilename string, content string, mode os.FileMode) error {
+	command := fmt.Sprintf("sudo install -m 0%o /dev/null %s && cat <<EOF | sudo tee %s\n%s\nEOF", mode, destFilename, destFilename, content)
+	_, err := runner.RunPrivate(command)
```

Comment:
Any specific reason we need to run those in private, since these command then will not be part of debug output.

Reply:
If you look at the 4 locations using `SetTextContentAsRoot`, 3 of them were using `RunPrivate` even if for 2 of these, it may have been a copy and paste mistake. With the debug log that we are copying a file to the VM, it's probably good enough.
If we want to dump the ssh command when possible, we can decide whether to use RunPrivate or not according to `mode` (`0?00` -> private, otherwise, not private).